### DISCUSSION TITLE
Setting up config.yaml to toggle between directories

### DISCUSSION THREAD

**Discussion by bonyboy** [beetbox/beets#3915] [Q&A] [answered]
I'm trying to figure out how to setup my config.yaml to have multiple directories to save files to and the CLI command to toggle between the two.  I've looked at [the docs](https://beets.readthedocs.io/en/stable/reference/config.html?highlight=BEETSDIR#environment-variable) and think I'm in the right place, but not sure about the command.

Or do I build 2 config.yaml files, each with 1 directory and toggle between them in the CLI?

Thanks for the help!

**Comment by sampsyo** [comment, marked as answer]
Yes, you can use `BEETSDIR` to change your configuration directory—but I actually don't think that's what you want. It sounds like you want to change the `directory:` configuration option within your beets configuration, right?

For that, you could use the global `-d` CLI flag: https://beets.readthedocs.io/en/stable/reference/cli.html#global-flags

Or, if you have other options you want to change simultaneously, you could consider using `-c` to add an "overlay" YAML file with all the options you want to occasionally customize.

**Reply by bonyboy** [reply]
Yes, you're correct in your assessment.

Does that mean in my situation (using the -d flag solution you suggested) that I'd have 1 directory listed in my config.yaml file and if I wanted to change that directory, I would say it explicitly in the CLI command and that command would override the directory listed in config.yaml?

**Reply by sampsyo** [reply]
Yes.
